packages/components/src/tag/Tag.module.css

```
@@ -14,6 +14,20 @@
   &[data-pressed] {
     background-color: --field-active-01;
   }
+
+  &.success[data-hovered],
+  .warning[data-hovered],
```

```suggestion
  &.warning[data-hovered],
```
Tror du skulle behöva göra `&` framför varje selector här

---

fixat